frontend/mobilityai_react/App/Screens/NewPatientForm.js

```diff
@@ -97,6 +98,15 @@ export default class PatientForm extends React.Component {
         this.setState({ onsetOfCondition: newText });
     }
 
+    toTwentyFourHourTime(txt){
+        const intRep = parseInt(txt, 10) || -1
+        if (intRep > 23 || intRep < 0) {
+            return "";
```

Comment:
We should turn the respective `Field` input box red and display a helpful message as to why their input is wrong instead of just deleting it.